modules/system/classes/CompilableAssets.php

```diff
@@ -0,0 +1,340 @@
+<?php namespace System\Classes;
+
+use Cms\Classes\Theme;
+use System\Classes\PluginManager;
+use Winter\Storm\Exception\SystemException;
+use Winter\Storm\Filesystem\PathResolver;
+use Winter\Storm\Support\Facades\Config;
+use Winter\Storm\Support\Facades\File;
+use Winter\Storm\Support\Str;
+
+/**
+ * Compilable assets using for Node.js compilation and processing.
+ *
+ * This works similar to the `System\Classes\CombineAssets` class in that it allows modules, plugins and themes to
+ * register configurations that will be passed on to Node.js based compilers for processing.
+ *
+ * @package winter\wn-system-module
+ * @author Ben Thomson <git@alfreido.com>, Jack Wilkinson <me@jackwilky.com>
```

Comment:
This is all you, bro - I ain't had anything to do with it :P
```suggestion
 * @author Jack Wilkinson <me@jackwilky.com>
```